universum.cabal

```diff
@@ -1,5 +1,5 @@
 name:                universum
-version:             0.7.2
+version:             0.7.3
```

Comment:
I think it should be 0.8.0, because it's a breaking change, strictly speaking (code which uses type applications with `show` will stop compiling).